sympy/physics/_biomechanics/musculotendon.py

```diff
@@ -214,10 +250,78 @@ def with_defaults(
         tendon_slack_length=None,
         peak_isometric_force=None,
         optimal_fiber_length=None,
+        v_M_max=Integer(10),
+        alpha_opt=Integer(0),
```

Comment:
curious why these two would be Float()?

Reply:
They are `Integer` here. Do you mean "why _aren't_ these two `Float`s"?

Reply:
Yes, sorry mistype.

A rational can represent any floating point, so I can see using that, but  when code generating an integer it will use the int type in the generated language. Right? Don't we want floats to be generated?

Reply:
You make a good point here. As these can take floating point values, then we should use `Float` here. There are also instances of this in both the `activation` and `curve` modules. I'll also make the same changes in those.

Reply:
Ok, thanks for checking into it.